src/main/java/com/thealgorithms/maths/MatrixRank.java

```diff
@@ -0,0 +1,62 @@
+package com.thealgorithms.maths;
+
+/**
+ *
+ *
+ * <h1>Rank of Matrix</h1>
+ *
+ * <p>
+ * The rank of a matrix is the largest number of linearly independent rows/columns of the matrix.
+ * The rank is not only defined for square matrices.
```

Comment:
This sentence is very confusing. And it does not provide much information.

Reply:
changed